src/com/google/javascript/rhino/JSDocInfo.java

```diff
@@ -223,6 +224,21 @@ private int getMaskForBitIndex(int bitIndex) {
    * with a string.
    */
   public static class StringPosition extends SourcePosition<String> {
+    static boolean areEquivalent(StringPosition p1, StringPosition p2) {
```

Comment:
Why static areEquivalent methods instead of a non-static equals()? Then it seems like we could have the equals() method in SourcePosition and not have to duplicate as much.